src/recordlinker/schemas/pii.py

```diff
@@ -347,6 +348,22 @@ def validate_and_normalize_telecom(self) -> typing.Self:
 
         return self
 
+    def normalized_value(self) -> str:
```

Comment:
@m-goggins I created this function to DRY up some code in the field_iter, basically the logic we were using for parsing TELECOM, PHONE and EMAIL.  While doing this, I was wondering, should we?  That is, should we just be normalizing the phone number at ingestion and storing the national number at that point, so we don't have to run `phonenumbers.parse(self.value).national_number` in the field_iter?  What do you think?